mozilla-release/config/version_win.pl

```diff
@@ -274,11 +274,11 @@ sub getNextEntry
 
 my $versionlevel=0;
 my $insideversion=0;
-if (open(RCINCLUDE, "<$rcinclude")) 
+if (open(RCINCLUDE, "<$rcinclude"))
 {
 	print RCFILE "// From included resource $rcinclude\n";
 #	my $mstring="";
-	while (<RCINCLUDE>) 
+	while (<RCINCLUDE>)
```

Comment:
for sake of version control (git blame, etc) we should avoid cutting trailing spaces. What you think @alver-cliqz ?